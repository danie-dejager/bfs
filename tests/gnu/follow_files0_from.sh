invoke_bfs links -print0 | bfs_diff -follow -files0-from -
