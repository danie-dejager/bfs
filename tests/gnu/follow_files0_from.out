links
links/broken
links/broken
links/deeply
links/deeply
links/deeply/nested
links/deeply/nested
links/deeply/nested
links/deeply/nested/broken
links/deeply/nested/broken
links/deeply/nested/broken
links/deeply/nested/broken
links/deeply/nested/dir
links/deeply/nested/dir
links/deeply/nested/dir
links/deeply/nested/dir
links/deeply/nested/file
links/deeply/nested/file
links/deeply/nested/file
links/deeply/nested/file
links/deeply/nested/link
links/deeply/nested/link
links/deeply/nested/link
links/deeply/nested/link
links/file
links/file
links/hardlink
links/hardlink
links/notdir
links/notdir
links/skip
links/skip
links/skip/broken
links/skip/broken
links/skip/dir
links/skip/dir
links/skip/file
links/skip/file
links/skip/link
links/skip/link
links/symlink
links/symlink
